Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM helsinkitest/node:16-slim AS deployable
 # In Openshift $HOME would be / by default
 ENV HOME /app
 
-COPY ./.env .
+#COPY ./.env .
```

Comment:
Maybe just delete this line, I doubt it's ever going to be uncommented.

Reply:
I will leave this for the project to edit if the line is not needed.